### ISSUE TITLE
I don't know where to begin

### ISSUE THREAD

**Issue by zzzzero01** [inbasic/media-converter#9]
I've downloaded the youtube video and audio add on, then the Muxer.  I might be having problems with the FFmpeg add on, but I am completely lost.  I'm not finding any clear step by step instructions.  If someone knows where I could find this, please point me in the right direction.  I have manually downloaded several FFMpeg files, and extracted one.  Other download prompts are unclear as to how I should be opening the file.  (which app to open with, windows 8).  

If anyone responds, please assume that I have to start from scratch.  There are way too many options in settings for the add on, and the muxer and the FFMpeg downloads to make any sense to me.

thnx

**Comment by inbasic** [comment]
Well, these two extensions are for different purposes.

Media Converter is for your media conversion needs. Though FFmpeg is required for both extensions to work at their highest capability.

Let me know your OS and I'll point you the the right place to download the FFmpeg executable.

**Comment by techwizkid69** [comment]
It's easy. you just have to open up console command and copy paste a series of commands, once you get to phase 3 or 4 of the installation process., I forget off the top of my head( perhaps node compile/install.js or something like that.) maybe , for its automation to bypass the inefficiency. Be sure to not use the buggy openeer as it can contain adware.

**Comment by inbasic** [comment]
@techwizkid69 

> once you get to phase 3 or 4 of the installation process

What installation process? It is simply one step!!

**Comment by techwizkid69** [comment]
@inbasic 
What installation process? It is simply one step!

Nuuuu, if it was simply 1 step, then this guy and many others wouldn't be having this same problem. You need IT skills to tackle it.

**Comment by Juan1to** [comment]
I totally agree with @inbasic. It's not that straight forward like one step and you're done. It does requires IT skills yes for me particularly and apparently also for our dear coleague zzzzero01 is like reading martian. I'm keeping the converter in the hope that I get to see the light but I'm uninstalling the ffmpeg. I will reinstall it in case I need it later.

**Comment by inbasic** [comment]
I am not clear how we can make the installation process easier. For now when the extension is installed, if offer FFmpeg installation which is automatic. When FFmpeg installation is over, the extension is ready to use. Let me know if you guys have better approach in mind.

@Juan1to FFmpeg is a must have component for MC to work.
